pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -47,10 +47,18 @@
         %div
           %a{href: "/break"}
             %img{src: "images/homepage/learn_more_drawn_button.png", style: "width: 250px; "}
+      - elsif entry[:type] == "code_break_check"
+        %div
+          %a{href: "/break"}
+            %img{src: "images/homepage/codebreak_check_button.png", style: "width: 250px; "}
+      - elsif entry[:type] == "code_break_home"
+        %div
+          %a{href: "/athome"}
+            %img{src: "images/homepage/codebreak_home_button.png", style: "width: 250px; margin-top: 10px"}
       - elsif entry[:type] == "code_break_video"
         %div
           %a{onclick: "return showVideo_mainvideo();", style: "cursor: pointer"}
-            %img{src: "images/homepage/watch_video_drawn_button.png", style: "width: 250px; margin-top: 10px; "}
+            %img{src: "images/homepage/watch_video_drawn_button.png", style: "width: 250px; margin-top: 10px"}
```

Comment:
keeping this around in case we need it again even though it's not currently in use?

Reply:
For now, yes.  Also I forgot :)